src/toolkits/drawing_classifier/dc_data_iterator.cpp

```diff
@@ -211,6 +214,10 @@ data_iterator::batch simple_data_iterator::next_batch(size_t batch_size) {
     batch_drawings.erase(pend, batch_drawings.end());
   }
 
+  // do noramization on each pixel
+  std::for_each(batch_drawings.begin(), batch_drawings.end(),
+                [=](float& x) { x /= scale_factor_; });
```

Comment:
If we use the term "scale" let's multiply here and initialize to 1.f / 255.f

Reply:
okay, doable.